pkg/adaptor/mongodb/session_test.go

```diff
@@ -0,0 +1,23 @@
+package mongodb
+
+import (
+	"testing"
+
+	mgo "gopkg.in/mgo.v2"
+)
+
+func TestClose(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping Close in short mode")
+	}
+	mgoSession, err := mgo.Dial(DefaultURI)
+	if err != nil {
+		t.Fatalf("unable to dial mongodb, %s\n", err)
+	}
+	mgoSession.DB("transporter_test").DropDatabase()
+	s, err := &Session{mgoSession}, nil
+	if err != nil {
+		t.Fatalf("unable to dial mongodb, %s\n", err)
+	}
+	s.Close()
```

Comment:
hrm... good point, it was primarily added for coverage but I'll see what can be done to verify it was properly closed.